docs/guide/custom-ui.md

```diff
@@ -250,6 +250,79 @@ Do not use core actions as your `action` property value - this would lead to unp
 * For backward compatibility, this method can also be called with the `moduleName` passed as the first argument and the object as the second, but this is discouraged.
 :::
 
+## Toggling the admin-bar visibility
+
+There are times when you want to allow people that don't have editing or content creation permissions to log into your project site. In that case, it may be desirable to not display the admin-bar. The `@apostrophecms/admin-bar` module `getShowAdminBar()` method can be extended to return `false` which will hide the admin-bar for that role when logged in.
```

Comment:
This is a great example, so I think you should add a note about how useful this is right of the box: you can set the "visibility" of pages and pieces to "login required" in order to restrict access to users with at least a guest account, without showing them the Apostrophe admin bar.